### ISSUE TITLE
What are the inputs for the training and testing phases

### ISSUE THREAD

**Issue by malinjie-hub** [NVlabs/nvdiffrec#57]
What parts of the input in the training phase are the multi-view images, camera poses and masks corresponding to multi-views? 
Will a .obj file be generated after training as a model for testing?
What are the parts of the input in the testing phase?
Thank you very much for your reply, urgent!

**Comment by jmunkberg** [comment]
> What parts of the input in the training phase are the multi-view images, camera poses and masks corresponding to multi-views?

See the datareaders, e.g., https://github.com/NVlabs/nvdiffrec/blob/main/dataset/dataset_nerf.py
to see how we extract the multi-view images, poses and masks from the datasets. Our datasets follows the torch.utils.data.Dataset interface

> Will a .obj file be generated after training as a model for testing?
Yes.

> What are the parts of the input in the testing phase?
A set of test images, masks and poses.

**Comment by jmunkberg** [comment]
Foreground segmentation masks: a gray-scale image indicating what pixels are covered by the object we want to reconstruct. In the NeRF synthetic dataset, this mask is stored in the alpha channel of the png files.

**Comment by malinjie-hub** [comment]
What does 'camera_angle_x' refer to? What does "rotation" mean? Why are there r_*_depth_0000.png and r_*_normal_0000.png images in the test folder of the nerf_synthetic dataset, and what are their functions? Why isn't it in the train and val folders?

**Comment by sadexcavator** [comment]
> 

I have the same question!

**Comment by sadexcavator** [comment]
> What does 'camera_angle_x' refer to? What does "rotation" mean? Why are there r___depth_0000.png and r___normal_0000.png images in the test folder of the nerf_synthetic dataset, and what are their functions? Why isn't it in the train and val folders?

same question for me here! looking forward to the reply~
